events.h

```diff
@@ -202,6 +204,36 @@ char EVT_start_loop(EVTHandler *handler);
  */
 void EVT_exit_loop(EVTHandler *handler);
 
+char EVT_enable_virt(EVTHandler *ctx, struct timeval *initTime);
+
+struct VirtClkState *EVT_clk(EVTHandler *ctx);
+
+/**
+ * Get the system's current absolute GMT time.
+ *
+ * @param tv A pointer to the timeval structure where the time gets stored
+ *
+ */
+int EVT_get_gmt_time(EVTHandler *ctx, struct timeval *tv);
+int EVT_get_gmt_time_virt(struct timeval *tv);
+
+/**
+ * Get the system's current time since an unknown reference point.  This
+ *  increases monotonically despite any changes to the system's current
+ *  understanding of GMT.
+ *
+ * @param tv A pointer to the timeval structure where the time gets stored
+ *
+ */
+#ifdef __APPLE__
```

Comment:
Shouldn't this logic me moved to the function(s) in eventTimer.c now?

Reply:
Yes, I agree. I will make that change